### PR TITLE
Copy editing

### PR DESCRIPTION
**Pull request by craigbox**
Punch things up and correct some missteaks.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2021/patch-tuesdays/index.md
Comment (nrjpoddar) [inline content/en/blog/2021/patch-tuesdays/index.md:23]:
```suggestion
upgrade issues users have seen. The Product Security Working Group intends to help Day 2 operations by having routine 
```

**Review by nrjpoddar — COMMENTED** [review]
cc @jacob-delgado

**Review by jacob-delgado — APPROVED** [review]

**Comment by ericvn** [comment]
LGTM. I'll approve after Jacob has approved.

**Review by ericvn — APPROVED** [review]

**Pull request merged**
